src/zenml/cli/stack_components.py

```diff
@@ -135,7 +138,7 @@ def describe_stack_component_command(name: Optional[str]) -> None:
             component_type, plural=True
         )
         repo = Repository()
-        components = repo.get_stack_components(component_type)
+        components = repo.zen_store.get_stack_components(component_type)
```

Comment:
why is this change needed ? Ideally, the Repository should act as a facade for all stacks/component related operations. From a design related perspective, the zen_store can be thought of as a back-end implementation detail that users shouldn't be exposed to.

Reply:
Tried to reflect the answer in the comment below. Let me know if you have further questions.